src/flag_gems/utils/libentry.py

```diff
@@ -93,6 +93,7 @@ def preload(self):
                 kwargs[k] = eval(v)
             for k, v in cfg_ls[-attrs:]:
                 numargs[k] = eval(v)
+            numargs.pop("enable_persistent", None)
```

Comment:
Could you add a comment line here describing this config option for reference?

Reply:
done